exercise/part2-abstracting-with-functions/index.js

```diff
@@ -109,11 +109,62 @@ L.tileLayer('http://stamen-tiles-{s}.a.ssl.fastly.net/toner-lite/{z}/{x}/{y}.{ex
 /* =====================
 
 Start code
+L.marker([39.95518350574191, -75.19931575943262]).bindTooltip('Coney Shack').addTo(foodMap);
+L.marker([39.95522447448491, -75.1998482020107]).bindTooltip('The Halal Guys').addTo(foodMap);
+L.marker([39.954899042422674, -75.20009627158234]).bindTooltip('Beilers Doughnuts').addTo(foodMap);
 
 ===================== */
 
-L.marker([39.9522, -75.1639]).bindTooltip('My Location').addTo(foodMap);
+/*
+
+const FoodIcon = L.icon({
+    iconUrl: 'foodicon.png',
+    iconSize: [100, 100], // size of the icon
+    iconAnchor: [500, 1126], //// point of the icon which will correspond to marker's location
+    popupAnchor: [-30, -30], // point from which the popup should open relative to the iconAnchor
+});
+
+const addPlace(map, lat, lng, name) => {
+  L.marker([lat, lng], { icon: FoodIcon }).bindTooltip(name).addTo(map);
+};
+
+addPlace(foodMap,39.95518350574191, -75.19931575943262,'Coney Shack');
+*/
+
+const FoodIcon = L.icon({
+  iconUrl: 'foodicon.png',
+  iconSize: [50, 50], // size of the icon
+  iconAnchor: [40, 40], // point of the icon which will correspond to marker's location
+  popupAnchor: [-30, -30], // point from which the popup should open relative to the iconAnchor
+});
+
+const PlayIcon = L.icon({
+  iconUrl: 'playicon.jpg',
+  iconSize: [50, 50], // size of the icon
+  iconAnchor: [40, 40], // point of the icon which will correspond to marker's location
+  popupAnchor: [-30, -30], // point from which the popup should open relative to the iconAnchor
+});
+
+const iconall = [FoodIcon, PlayIcon];
+
+let addPlace = (map, lat, lng, i, name) => {
```

Comment:
@yyebei The only reason the tests are failing is because they expect the `name` argument to come before the `i` argument. But I think that's ok.